interop-cats/shared/src/main/scala/zio/interop/queue/CQueue.scala

```diff
@@ -0,0 +1,72 @@
+package zio.interop.queue
```

Comment:
I think this can just be in the `cats.interop` package like the `Schedule` compatibility implementation to mirror the structure in ZIO.

Reply:
agreed 👍